niji/main.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
+
+// var appRoutes = require('./routes/app');
+var apiRoutes = require('./routes/api');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'app/prod/hbs'));
+app.set('view engine', 'hbs');
+
+// app.use(favicon(path.join(__dirname, 'app/prod/img', 'favicon.ico')));
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'app/prod/')));
+app.use(passport.initialize());
+app.use(passport.session());
+
+app.use(function(req, res, next) {
```

Comment:
What's the purpose of this? Not that I don't like it, I just don't know what it is.